### PR TITLE
New Code Editor (not based on Runestone)

### PR DESCRIPTION
**Pull request by pal03377**

### CODE REVIEW

File: Themis/Views/Assessment/CodeEditor/CodeView.swift
Comment (AlexExcellence) [inline Themis/Views/Assessment/CodeEditor/CodeView.swift]: We should remove this by implementing the lasso toggle functionality. But merging is more important for now.

**Review by AlexExcellence — APPROVED** [review]

File: Themis/Views/Assessment/AssessmentView.swift
Comment (AlexExcellence) [inline Themis/Views/Assessment/AssessmentView.swift]: This shouldn't have been removed!

File: Themis/Views/Assessment/AssessmentView.swift
Comment (AlexExcellence) [inline Themis/Views/Assessment/AssessmentView.swift]: This can be removed now, since we implemented the custom feedback Action in EditMenu

**Review by AlexExcellence — CHANGES_REQUESTED** [review]

**Review by tomrudnick — APPROVED** [review]

**Review by huber-florian — DISMISSED** [review]
Please rebase/merge with current develop and afterwards every reviewer should have a look again.

**Comment by AlexExcellence** [comment]
> Please rebase/merge with current develop and afterwards every reviewer should have a look again.

It is already up to date!

**Comment by AlexExcellence** [comment]
Just noticed, that inline feedbacks highlights aren't deleted anymore when deleting the feedback (Even on current develop branch). This is because the file property of the AssessmentFeedback is always nil, which wasn't the case before. Will fix this and after that it is ready to be merged.

**Review by AlexExcellence — APPROVED** [review]

**Pull request merged**
